### ISSUE TITLE
[BUG] - Windows Environment Variables doesn't work

### ISSUE THREAD

**Issue by Konstantine-Nevzhunskyi** [alefragnani/vscode-project-manager#584]
Providing rootPath in projects.json with some Environment Variable doesn't work.

- Extension version: 12.2.0
- VSCode version: 1.63.2
- OS version: Windows 11

**Steps to reproduce**

1. Set rootPath like "rootPath": "%GD_Path%\\\test"
2. Try to open project.
3. Get error "Path does not exist: The path '\\%GD_Path%\test' does not exist on this computer."

**Comment by alefragnani** [comment]
Hi @Konstantine-Nevzhunskyi ,

The extension does not support environment variable. You can, however, use `~` or `$home` as a replacement for your _home folder_. 

There is an open issue (#550) requesting to add support for other/environment variables. Feel free to add new comments and upvote the issue, so I’m able to track user feedback.

Hope this helps

**Comment by Konstantine-Nevzhunskyi** [comment]
Sorry, I just thouhgt that it is bug, coz I just tried to use usual path for my system. In issue #550 talks about more powerfull functionality with some parsing/replacement in pathes. So I decided that there is some bug (for example some path transformation regardless of the fact that there are pathes in Windows that starts with "%"), as I know there is no need in additional code to make this pathes right handled by system.
